runtime/src/cost_tracker.rs

```diff
@@ -95,6 +95,29 @@ impl CostTracker {
         Ok(self.block_cost)
     }
 
+    pub fn update_execution_cost(
+        &mut self,
+        estimated_tx_cost: &TransactionCost,
+        actual_execution_units: u64,
+    ) {
+        let estimated_execution_units = estimated_tx_cost.bpf_execution_cost;
+        if actual_execution_units == estimated_execution_units {
```

Comment:
nit: You can cover all cases with the [`Ordering`](https://doc.rust-lang.org/std/cmp/enum.Ordering.html) enum

Reply:
Good call, [commit](5859d34171a34f4c1bf4d1fc04d8e633f9fda4a1) for it